pkg/core/batch.go

```diff
@@ -14,11 +14,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package fftypes
+package core
```

Comment:
Why rename from fftypes to core? Is it just to differentiate it from the base fftypes in common?

Reply:
Yep, lots of files pull in both - so this avoids the need to alias.